packages/calypso-e2e/src/lib/blocks/block-flows/layout-grid.ts

```diff
@@ -50,6 +50,16 @@ export class LayoutGridBlockFlow implements BlockFlow {
 		const twoColumnButtonLocator = context.editorLocator.locator( selectors.twoColumnButton );
 		await twoColumnButtonLocator.click();
 
+		/**
+		 * Workaround for the issue where the inline inserter of the Columns
+		 * block disappears when the "+" button is clicked too early after a
+		 * column count has been selected. This issue is not reproducible
+		 * manually (unless you're ⚡️ fast) and happens only with the testing
+		 * framework.
+		 */
+		const page = twoColumnButtonLocator.page();
+		await page.evaluate( () => new Promise( ( r ) => setTimeout( r, 1000 ) ) );
```

Comment:
**comment/non-blocking**: While it's usually not recommended, I personally would suggest using `page.waitForTimeout()` instead. 

With that said though, is it not possible to wait for an attribute such as `.is-loading` or `.placeholder` to disappear instead? I recall that earlier in the framework's development we debated the merits of waiting for timeouts vs waiting for a selector to disappear.

Reply:
> While it's usually not recommended, I personally would suggest using page.waitForTimeout() instead.

Ah, forgot about this one. Thanks! 👍 

> With that said though, is it not possible to wait for an attribute such as .is-loading or .placeholder to disappear instead? I recall that earlier in the framework's development we debated the merits of waiting for timeouts vs waiting for a selector to disappear.

I couldn't really find any loading indication that we could hook on to at this point. 😞  The issue seems to be a rerender that's happening right after selecting the number of columns for the Columns block. Playwright goes straight to clicking the `+` button to open an inline inserter (rendered internally via Dropdown within the block AFAIR), and soon after that (<500ms), a rerender is triggered, and the inline Inserter is being wiped out, breaking the flow in consequence. As mentioned earlier, this is happening only because the simulation can go much faster than a human would. On that note, since it doesn't hurt our users, my suggestion would be to go with the fixed delay monkey patch for the Gutenberg v14.2.0 upgrade and I'll keep looking for proper handling of this issue. 💪 /cc @fullofcaffeine

Reply:
Created an issue for this https://github.com/Automattic/block-experiments/issues/294